src/vcProject.cpp

```diff
@@ -833,3 +837,101 @@ void vcProject_ExtractAttributionText(udProjectNode *pFolderNode, const char **p
+      for (const char *pStr : pProgramState->projectInfoTextures.infoStrings)
+        udFree(pStr);
+      pProgramState->projectInfoTextures.infoStrings.Clear();
+      pProgramState->projectInfoTextures.textures.Clear();
+      pProgramState->projectInfoTextures.textureSizes.Clear();
+
+      const size_t invalidLen = udStrlen(pInfo);
+
+      // Discover Images
+      int firstIndex = 0;
+      int startStrIndex = 0;
+      while (true)
+      {
+        size_t startImagePos = 0;
+        size_t openBracketPos = 0;
+        size_t closeBracketPos = 0;
+        size_t openParenthesisPos = 0;
+        size_t closeParenthesisPos = 0;
+
+        if (udStrchr(pInfo + firstIndex, "!", &startImagePos) == nullptr)
+          break;
+        startImagePos += firstIndex;
+        if (udStrchr(pInfo + startImagePos, "[", &openBracketPos) == nullptr)
+          break;
+        openBracketPos += startImagePos;
+        if (udStrchr(pInfo + openBracketPos, "]", &closeBracketPos) == nullptr)
+          break;
+        closeBracketPos += openBracketPos;
+        if (udStrchr(pInfo + closeBracketPos, "(", &openParenthesisPos) == nullptr)
+          break;
+        openParenthesisPos += closeBracketPos;
+        if (udStrchr(pInfo + openParenthesisPos, ")", &closeParenthesisPos) == nullptr)
+          break;
+        closeParenthesisPos += openParenthesisPos;
+
+        if (startImagePos == invalidLen ||
+          openBracketPos == invalidLen ||
+          closeBracketPos == invalidLen ||
+          openParenthesisPos == invalidLen ||
+          closeParenthesisPos == invalidLen)
+          break;
+
+        size_t lastCharPos = closeParenthesisPos;
+        firstIndex = (int)(lastCharPos + 1);
+
+        if (openBracketPos == startImagePos + 1 &&
+          closeBracketPos > openBracketPos &&
+          openParenthesisPos == closeBracketPos + 1 &&
+          closeParenthesisPos > openParenthesisPos)
+        {
+          // Text
+          size_t prevStrLen = startImagePos - startStrIndex;
+          if (prevStrLen > 0)
+          {
+            const char *pSubStr = udStrndup(pInfo + startStrIndex, prevStrLen);
+            pProgramState->projectInfoTextures.infoStrings.PushBack(pSubStr);
+          }
+
+          // Texture Alt Text
+          if (closeBracketPos == openBracketPos + 1)
+          {
+            pProgramState->projectInfoTextures.textureAltStrings.PushBack((const char *)nullptr);
+          }
+          else
+          {
+            const char *pSubStr = udStrndup(pInfo + openBracketPos + 1, closeBracketPos - openBracketPos - 1);
+            pProgramState->projectInfoTextures.textureAltStrings.PushBack(pSubStr);
+          }
+
+          // Texture
+          const char *pTexPath = udStrndup(pInfo + (openParenthesisPos + 1), closeParenthesisPos - openParenthesisPos - 1);
+          uint32_t w, h;
+          vcTexture *pNewTexture = nullptr;
+          vcTexture_CreateFromFilename(&pNewTexture, pTexPath, &w, &h);
```

Comment:
This will probably need to end up being one of the async functions for two reasons:
- If this is on the main thread, it's going to stall performing the HTTP request for the image
- If this isn't on the main thread, the GPU texture handle will be created outside of the main thread